guide/14-deep-learning/object-detection.ipynb

```diff
@@ -16,7 +16,18 @@
    },
```

Comment:
<p>Hi @yongyao, I think we should clarify that exporting the training data and inferencing can be done using both ArcGIS Pro and ArcGIS Image Server. We should explain how the data can be exported using ArcGIS Pro using the Export Training Data tool. Those who have used ArcGIS Pro to export the training samples should be guided straight to the Model Training section. </p><p> </p><p>Also, in the Deployment section, we should add documentation stating that the saved model can be used in ArcGIS Pro or Image Server. For those using ArcGIS Pro, we should explain how they can use the Detect Object or Classify Pixels tools. We should then tell about the ability to scale out the inferencing using distributed raster analytics and Image Server and guide users through the subsequent sections.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/474/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>